composer.json

```diff
@@ -0,0 +1,5 @@
+{
+"autoload": {
```

Comment:
Rest of parameters explaining application are missing. Refer to composer format. https://getcomposer.org/doc/04-schema.md